code/modules/asset_cache/transports/asset_transport.dm

```diff
@@ -0,0 +1,153 @@
+/// When sending mutiple assets, how many before we give the client a quaint little sending resources message
+#define ASSET_CACHE_TELL_CLIENT_AMOUNT 8
+
+/// Base browse_rsc asset transport
+/datum/asset_transport
+	var/name = "Simple browse_rsc asset transport"
+	var/static/list/preload
```

Comment:
is there any particular reason to use `static` here? You change `preload` everytime you initialize a new asset_transport datum which kind of defeats the purpose of having a shared list between all types.

Reply:
I'm unsure on the exact specifics of how statics work, but I'm pretty sure the `/datum/asset_transport` should only be created twice in any given round.